casanovo/data/ms_io.py

```diff
@@ -143,10 +230,95 @@ def set_ms_run(self, peak_filenames: List[str]) -> None:
             )
             self._run_map[filename] = i
 
+    def get_report_dict(self) -> Optional[Dict]:
+        """
+        Generate sequencing run report
+
+        Parameters
+        ----------
+            score_bins: List[float], Optional
+                Confidence scores for creating confidence CMF, see getScoreBins
+
+        Returns:
+            report_gen: Dict
+                Generated report represented as a dictionary, or None if no
+                sequencing predictions were logged
+        """
+        results_table = DataFrame(
```

Comment:
**nitpick:** No need really to convert this to a dataframe, you can just use two lists instead.